app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java

```diff
@@ -97,6 +105,39 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 openChart.putExtra(EVENT_ID, initialEventId);
                 startActivity(openChart);
             });
+
+        SharedPreferences sharedPreferences = getActivity().getSharedPreferences(Constants.FOSS_PREFS, MODE_PRIVATE);
+
+        if (sharedPreferences.getBoolean(getString(R.string.developer_mode_key), false)) {
```

Comment:
Move the logic from view to presenter/VM

Reply:
Done.